pci.ids

```diff
@@ -96,6 +102,7 @@
 	51a3  7300 Dual Port U.2 MAX
 144d  Samsung Electronics Co Ltd
 	a808  NVMe SSD Controller SM981/PM981/PM983
+	a80a  NVMe SSD Controller PM9A1/980PRO
 		1d49 403b  Thinksystem U.2 PM983 NVMe SSD
```

Comment:
now "1d49 403b  Thinksystem U.2 PM983 NVMe SSD" become subsystem of a80a instead of a808. Is it intentional?

Reply:
It's my mistake, need make "1d49 403b  Thinksystem U.2 PM983 NVMe SSD" subsystem of a808